### PR TITLE
Added new query for AppArmor Config

### LINKED ISSUE #632: Add app_armor_config query for kubernetes
### Platform
kubernetes

### Description
Containers should be configured with AppArmor for any application to reduce its potential attack

### PR DESCRIPTION
**Pull request by soniantao**
Closes #632

### CODE REVIEW

File: assets/queries/k8s/app_armor_config/metadata.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "id": "app_armor_config",
+  "queryName": "app_armor_config",
```
Comment (Ruben-Silva) [inline assets/queries/k8s/app_armor_config/metadata.json:3]: The query name and id should explain the vulnerability. Try something like "Missing_App_Armor_Config"

File: assets/queries/k8s/app_armor_config/metadata.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  "severity": "LOW",
+  "category": null,
+  "descriptionText": "Containers should be configured with AppArmor for any application to reduce its potential attack",
+  "descriptionUrl": "https://gsl.dome9.com/D9.K8S.IAM.23.html"
```
Comment (Ruben-Silva) [inline assets/queries/k8s/app_armor_config/metadata.json:7]: The URL should point to official kubernetes documentation. Use this one "https://kubernetes.io/docs/tutorials/clusters/apparmor/".

File: assets/queries/k8s/app_armor_config/test/positive_expected_result.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+[
+	{
+		"queryName": "app_armor_config",
```
Comment (Ruben-Silva) [inline assets/queries/k8s/app_armor_config/test/positive_expected_result.json:3]: The query name should explain the vulnerability. Try something like "Missing_App_Armor_Config"

File: assets/queries/k8s/app_armor_config/test/positive_expected_result.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+		"severity": "LOW",
+		"line": 5
+	},
+	{
+		"queryName": "app_armor_config",
```
Comment (Ruben-Silva) [inline assets/queries/k8s/app_armor_config/test/positive_expected_result.json:8]: The query name should explain the vulnerability. Try something like "Missing_App_Armor_Config"

File: assets/queries/k8s/app_armor_config/test/positive_expected_result.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+		"severity": "LOW",
+		"line": 27
+	},
+	{
+		"queryName": "app_armor_config",
```
Comment (Ruben-Silva) [inline assets/queries/k8s/app_armor_config/test/positive_expected_result.json:13]: The query name should explain the vulnerability. Try something like "Missing_App_Armor_Config"

File: assets/queries/k8s/app_armor_config/test/positive_expected_result.json [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+		"severity": "LOW",
+		"line": 39
+	},
+	{
+		"queryName": "app_armor_config",
```
Comment (Ruben-Silva) [inline assets/queries/k8s/app_armor_config/test/positive_expected_result.json:18]: The query name should explain the vulnerability. Try something like "Missing_App_Armor_Config"

**Review by Ruben-Silva — CHANGES_REQUESTED** [review]

File: assets/queries/k8s/app_armor_config/metadata.json (hunk 1 above)
Comment (soniantao) [reply assets/queries/k8s/app_armor_config/metadata.json:3]: You are correct Ruben! I'll change it per suggestion

**Pull request merged**